dev-haskell/regex-base/Manifest

```diff
@@ -1,2 +1 @@
-DIST regex-base-0.93.2.tar.gz 9156 BLAKE2B 160f749c5c5e4c9911c18a4ad3dbe46865c9872f419457387e249747c5aade2cd2afae1f80757d3c0788398f332ea243f35e826f77f220517480ae4b42928a21 SHA512 4f6269cb1c1fefa2164d999e2e787118edc5fd88e356535e50f143d0b98de4fa1854959955e183aed27a77335b81063a2385bb6bdeb8cb80e7fdd812ab96d22a
```

Comment:
It's hard to review package bumps when removal deletes more than one version. On top of that `repoman` check fails:

```
  dependency.bad [fatal]        2
   dev-haskell/yi-language/yi-language-0.2.1.ebuild: DEPEND: ~amd64(default/linux/amd64/17.1/no-multilib/hardened)
['<dev-haskell/regex-base-0.94:=[profile?]']
```

Reply:
If there's some way to make it easier for you, let me know. As far as this `yi-language` package, that version is super old and I'm confused what it's doing in the tree (current version is `0.19.0` according to cabal). I don't see anything listing this package as a dependency either. Can we just drop this version?

Reply:
I should have caught this, but it's good the CI caught it for me....

Should I have two separate commits for this: one for the bump and another to remove the old package(s)?

Reply:
Removing one ebuild and adding another is fine in one commit. Diffs are usually cleanest that way.

I'd say removing multiple (outdated) ebuilds should go in a separate commit. They are usually mechanical and don't require much scrutiny.

yi-language should be fine to be masked for removal.

Reply:
Ok, I'm rebasing now. I'll force push soon and see what the CI says.